cfgov/v1/blocks.py

```diff
@@ -109,7 +109,20 @@ class Feedback(AbstractFormBlock):
         required=False,
         default='Do you currently own a home?'
     )
-    button_text = blocks.CharBlock(default='Submit')
+    button_text = blocks.CharBlock(
+        default='Submit'
+    )
+    contact_advisory = blocks.RichTextBlock(
+        required=False,
+        default=('If you provide your contact information, '
+                 'we may contact you to clarify your question. '
+                 'This information is collected in accordance with the '
+                 '<a href="/owning-a-home/help-us-improve/'
+                 'paperwork-reduction-act-statement/">'
+                 'Paperwork Reduction Act Statement</a> and the '
+                 '<a href="/owning-a-home/help-us-improve/'
+                 'privacy-act-statement/">Privacy Act Statement</a>.')
```

Comment:
It would be nice not to have to hardcode these links here. Since this is a RichTextBlock, can editors internal links to these actual Wagtail pages? Actually, can we just remove this default text entirely and just require content editors to add it? How many pages does this apply to?